### DISCUSSION TITLE
Is there any rule to remove standby coordinator first not active coordinator during milvus upgrade?

### DISCUSSION THREAD

**Discussion by SeunghyunYoo** [milvus-io/milvus#29824] [Q&A and General discussion]
Is there any rule to remove standby coordinator first not active coordinator during milvus upgrade?

I can't find any codes/rule about coordinator remove order when active-standby mode is applied.

I think the standby coordinator have to be removed first, 
because if not `the old version standby coord` and `new version coord` have to race to get ACTIVE key and it induce several bad situations during live upgrade.

Plus, if there are code about this, would you reference the code for me?

**Comment by LoveEachDay** [comment]
@SeunghyunYoo Yes you are right. There's no guarantee for the order of rolling upgrade milvus coordinators.
This should be ok, we have the code compatibility for coordinators between the most recent two versions.
